src/structures/VoiceState.js

```diff
@@ -63,10 +64,22 @@ class VoiceState extends Base {
      */
     this.streaming = data.self_stream || false;
     /**
-     * The ID of the voice channel that this member is in
+     * The ID of the voice channel or stage channel that this member is in
      * @type {?Snowflake}
      */
     this.channelID = data.channel_id || null;
+    /**
+     * Whether this member is suppressed from speaking. This property is specific to stage channels only.
+     * @type {boolean}
+     */
+    this.suppress = data.suppress;
+    /**
```

Comment:
```suggestion
    this.suppress = data.suppress;

    /**
```

Reply:
yes very helpful change

Reply:
I'm just following the convention for the rest of the properties in that file